WcaOnRails/spec/mailers/competitions_mailer_spec.rb

```diff
@@ -66,6 +66,7 @@
       expect(mail.to).to eq ["delegates@worldcubeassociation.org"]
       expect(mail.cc).to match_array competition.delegates.pluck(:email)
       expect(mail.from).to eq ["notifications@worldcubeassociation.org"]
+      expect(mail.reply_to).to match_array (["delegates@worldcubeassociation.org"] + competition.delegates.pluck(:email))
```

Comment:
One way to to avoid parentheses and this `+`. It's personal thing what's more sweet of course.
```ruby
expect(mail.reply_to).to match_array ["delegates@worldcubeassociation.org", ...competition.delegates.pluck(:email)]
```